components/avatar/avatar.vue

```diff
@@ -143,21 +206,53 @@ export default {
     },
 
     isSvgType (element) {
-      return element?.tagName?.toUpperCase() === 'SVG';
+      return element?.tagName?.toUpperCase() === 'SVG' || element?.tag?.toUpperCase() === 'SVG';
     },
 
     isImageType (element) {
-      return element?.tagName?.toUpperCase() === 'IMG';
+      return element?.tagName?.toUpperCase() === 'IMG' || element?.tag?.toUpperCase() === 'IMG';
```

Comment:
same deal here

Reply:
rollbacked this change, with the refactor just using `element?.tagName?.toUpperCase() === 'IMG'` is enough